css/product.css

```diff
@@ -31,6 +31,16 @@
 .filter-category span {
   margin-left: 0.5rem;
```

Comment:
As Vikram suggested there is no need to change px to rem if its below 10px

Reply:
Got it